Dockerfile

```diff
@@ -26,7 +26,7 @@ ENV NODE_ENV=production \
 RUN yarn build
 
 # Backend builder stage
-FROM docker.io/library/golang:1.23.2-alpine3.20 as go-builder
+FROM docker.io/library/golang:1.23.4-alpine3.20 as go-builder
```

Comment:
Had to lower the alpine version to `3.20` as there is no `golang:1.23.4-alpine3.21` image, yet.

If I'm not mistaken we'd be fine using just `golang:1.23.4-alpine` here as it will always pick the latest alpine version available, see section "Simple Tags" at <https://hub.docker.com/_/golang>.

Reply:
maybe this is an option for the next release? @ntap-fge @ntap-nmarco